core/app/models/workarea/pricing/calculators/tax_calculator.rb

```diff
@@ -5,23 +5,50 @@ class TaxCalculator
         include Calculator
 
         def adjust
+          adjust_shipped_items_tax
+          adjust_not_shipped_items_tax
+        end
+
+        def adjust_shipped_items_tax
           shippings.each do |tmp_shipping|
             next unless tmp_shipping.address.present?
 
             adjustments_to_tax = price_adjustments_for(tmp_shipping)
-            TaxApplier.new(tmp_shipping, adjustments_to_tax).apply
+            ShippedTaxApplier.new(tmp_shipping, adjustments_to_tax).apply
```

Comment:
So this will break all current tax integrations? Why not make the digital items the new case so integrations can gracefully upgrade to include them?

Reply:
have to stop thinking about it as digital items, rather than "not shipped" items. I see your point though.

Reply:
Yeah, I'm not loving the language around this so far, but I'm not sure what a better word is. What kinds of products aren't shipped and aren't digital?